openmrs/startup.sh

```diff
@@ -17,15 +17,10 @@ create_tables=${DB_CREATE_TABLES}
 auto_update_database=${DB_AUTO_UPDATE}
 EOF
 
-cat > /usr/local/tomcat/.OpenMRS/openmrs-runtime.properties << EOF
-encryption.vector=KOWPyyXpnQfFWo2p2Cl8Zg\=\=
-connection.url=jdbc\:mysql\://${DB_HOST}\:3306/${DB_DATABASE}?autoReconnect\=true&sessionVariables\=default_storage_engine\=InnoDB&useUnicode\=true&characterEncoding\=UTF-8
-module.allow_web_admin=true
-connection.username=${DB_USERNAME}
-auto_update_database=true
-encryption.key=gj6a6U9+QjiuKn6oDA1l8g\=\=
-connection.driver_class=com.mysql.jdbc.Driver
-connection.password=${DB_PASSWORD}
+cat > /tmp/db_credentials.txt << EOF
```

Comment:
`/tmp/` does not sound like a safe place to put this file.
`/etc/mysql/` should do.
Maybe this could be renamed as `db-credentials.cnf` (`cnf` being a usual extension for MySQL configuration files)